dlt/destinations/impl/snowflake/snowflake.py

```diff
@@ -163,9 +163,17 @@ def __init__(
             from_clause = f"FROM {stage_file_path}"
 
         # decide on source format, stage_file_path will either be a local file or a bucket path
-        source_format = "( TYPE = 'JSON', BINARY_FORMAT = 'BASE64' )"
+        if file_name.endswith("jsonl"):
+            source_format = "( TYPE = 'JSON', BINARY_FORMAT = 'BASE64' )"
         if file_name.endswith("parquet"):
             source_format = "(TYPE = 'PARQUET', BINARY_AS_TEXT = FALSE, USE_LOGICAL_TYPE = TRUE)"
+        if file_name.endswith("csv"):
+            # empty strings are NULL, no data is NULL, missing columns (ERROR_ON_COLUMN_COUNT_MISMATCH) are NULL
```

Comment:
imho the info how this is handled should go into the docs (if it's not done in a later PR.)

Reply:
will do it now